viewer/app/src/github-api.js

```diff
@@ -151,7 +152,6 @@ export class GithubApi {
       // not return a 304 and so will be overwritten.
       return idbKeyval.set(id, response).then(_ => {
         logger.hide();
-        // @ts-expect-error - TODO(bckenny): tsc unable to flatten promise chain here
```

Comment:
not sure when the promise thing was fixed (it was similar to other cases where it ended up as `Promise<Promise<whatever>>` but this one was fixed sometime before 4.5), but this `@ts-expect-error` was actually suppressing the fact that tsc thought `response` could be `null` here.

It probably can't be--I don't think github would return a 304 for this request without an etag set--but if there's no `cachedGist` then falling through to the error case makes sense, so it's a natural fix.